feature/platform/ate_tests/telemetry_inventory_test/telemetry_inventory_test.go

```diff
@@ -490,10 +429,22 @@ func ValidateComponentState(t *testing.T, dut *ondatra.DUTDevice, cards []string
 		}
 
 		if p.mfgDateValidation {
-			mfgDate := gnmi.Get(t, dut, component.MfgDate().State())
-			t.Logf("Hardware card %s mfgDate: %s", card, mfgDate)
-			if mfgDate == "" {
-				t.Errorf("component.MfgName().Get(t) for %q): got empty string, want non-empty string", card)
+			if !(*deviations.SupervisorMfgDateUnsupported && p.pType == oc.PlatformTypes_OPENCONFIG_HARDWARE_COMPONENT_CONTROLLER_CARD) {
+				mfgDate := gnmi.Get(t, dut, component.MfgDate().State())
+				t.Logf("Hardware card %s mfgDate: %s", card, mfgDate)
+				if mfgDate == "" {
+					t.Errorf("component.MfgName().Get(t) for %q): got empty string, want non-empty string", card)
+				}
+			}
+		}
+
+		if p.fwVerValidation {
```

Comment:
same as above

Reply:
@vosipchu is there a Google bug capturing this conversation?

Reply:
LGTM for removal